index.html

```diff
@@ -7,12 +7,611 @@
+
+      <div class="black">
+        <a href="#">News</a>
+        <a href="#">Events</a>
+        <a href="#">Business</a>
+        <a href="#">AMAs</a>
+        <a href="#">Spaces</a>
+        <a href="#">Terms & Conditions</a>
+        <a href="#">About</a>
+        <a href="#">Advertise</a>
+        <a href="#">Jobs</a>
+        <a href="#">Contact</a>
+      </div>
+
+      <div class="white">
+        <a href="#">Latest</a>
+        <a href="#">Hard Fork</a>
+        <a href="#">Plugged</a>
+        <a href="#">readme</a>
+        <a href="#">growth quarters</a>
+        <a href="#">shift</a>
+        <a href="#">neural</a>
+      </div>
+
+      <div class="yellow">
+        <a href="#">Calling young digital in the Netherlands - Be recognized in the T500 </a>
+      </div>
+    </div>
+  </nav>
+</header>
+  <!-- #THE HEAD SECTION-->
+
+  <section class="head">
+
+    <div class="left-head top-right">
+      <h2 class="tnw">TNW</h2>
+      <a href="#"><span class="tnw-img"></span></a> 
+      <p class="date"> Tuesday -- January 26, 2021</p>
+      <a href="#" class="tnw-bottom">Opal cuts off your apps’ internet access to help you stop doomscrolling</a>
+      <a href="#" class="author">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago 12 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="top-right-head left-head">
+      <a href="#" class="tnw-bottom">Google unveils privacy-friendly AI cookie killer</a>
+      <a href="#" class="author">by Tomas David <i class="far fa-clock"></i> 40 minutes ago 8 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="bottom-right-head left-head">
+      <a href="#" class="tnw-bottom">This Mac app automatically detects your duplicate files</a>
+      <a href="#" class="author">by Ivan Mehta <i class="far fa-clock"></i> 2 hours ago 25 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+  </section>
+
+  <div class="center">
+    <!-- #THE LATEST SECTION
+
+  <section class="latest-container">
+    <a href="#">
+      <h2>Latest news</h2>
+    </a>
+
+    <div class="latest">
+      <div class="latest1">
+        <a href="#">Want to drive a hydrogen EV in the US? Move to California </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest2">
+        <a href="#">Help kill the ‘Bernie sitting’ meme with this easy-to-use image generator </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest3">
+        <a href="#">Uber expands its low-emission ride options to 1,400 more North American cities </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest4">
+        <a href="#">3 reasons why almost every VC investor passed on Zoom </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest5">
```

Comment:
Kindly remove all commented code, use git history to retrieve previous code

Reply:
The commented code is for the other sections of the website as we have structured the website and styled the shown sections as requested